scripts/import_from_biosamples.py

```diff
@@ -1120,15 +1160,23 @@ def import_into_es(data, index_prefix, my_type, es):
     for biosample_id in sorted(list(data.keys())):
         INDEXED_SAMPLES[biosample_id] = 1
         es_doc = data[biosample_id]
+        error_messages = list()
+        status = ''
         for ruleset in RULESETS:
             if validation_results[ruleset]['detail'][biosample_id]['status'] == 'error':
+                status = 'error'
+                error_messages.append(f"error\t{ruleset}\t"
+                                      f"{validation_results[ruleset]['detail'][biosample_id]['message']}")
                 logger.error(f"{biosample_id}\t{my_type}\terror\t"
                              f"{ruleset}\t{validation_results[ruleset]['detail'][biosample_id]['message']}")
```

Comment:
I think that we need to completely remove logger.* calls and change it on insert_es_log